qiita_db/support_files/patches/python_patches/38.py

```diff
@@ -0,0 +1,21 @@
+#
+# This is a really heavy step and it will be better not to run after each test
```

Comment:
Uncomment this. When this is executed in the tests, the database is empty, since it is populated after all patches are applied.